guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,25 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Linux Unified Key Setup (LUKS) during host provisioning.
+Optionally, you can also bind the disk to a Tang server through Clevis for decryption during boot.
+You must associate the partition table with your operating system entry.
+Then, you assign the partition table to your host group or select it manually during provisioning.
+
+Encrypted disks on hosts protect data at rest.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
+For more information, see xref:creating-partition-tables_{context}[].
+. Create a host group that uses the `{client-provisioning-template-type} default encrypted` partition table.
+For more information, see {ManagingHostsDocURL}Creating_a_Host_Group_managing-hosts[Creating a host group] in _{ManagingHostsDocTitle}_.
++
+* You either have to add the host parameter `disk_enc_passphrase` as type *string* and your cleartext passphrase of the LUKS container as the value, or, to use Clevis/Tang to decrypt the disk of your hosts during boot time, add the host parameter `disk_enc_tang_servers` as type *array* and your list of Tang servers (example: `["1.2.3.4"]` or `["{server-example-com}", "5.6.7.8"]`).
```

Comment:
This sentence is very long and not easy to read. I strongly recommend breaking this into separate list items.

Reply:
Reworded. Please recheck @apinnick